project/HmrcBuild.scala

```diff
@@ -47,10 +48,16 @@ object HmrcBuild extends Build {
         Resolver.bintrayRepo("hmrc", "releases"),
         "typesafe-releases" at "http://repo.typesafe.com/typesafe/releases/"
       ),
-      crossScalaVersions := Seq("2.11.7")
+      crossScalaVersions := Seq("2.11.7"),
+      testGrouping in sbt.Test := oneForkedJvmPerTest((definedTests in sbt.Test).value)
     )
     .settings(TwirlKeys.templateImports ++= Seq("play.api.mvc._", "play.api.data._", "play.api.i18n._", "play.api.templates.PlayMagic._"))
     .settings(unmanagedSourceDirectories in sbt.Compile += baseDirectory.value / "src/main/twirl")
+
+  private def oneForkedJvmPerTest(tests: Seq[TestDefinition]) =
```

Comment:
If you abstracted the html into a fragment you could then test it in a similar fashion to: https://github.com/hmrc/play-ui/blob/master/src/test/scala/uk/gov/hmrc/play/views/layouts/BetaBannerSpec.scala

Then you wouldn't need any of the new work in `project/HmrcBuild.scala`

### optimizelySnippet.html
```

@import uk.gov.hmrc.play.config.OptimizelyConfig._

@(idOpt: Option[String] = optimizelyProjectId)
  @idOpt.map { id =>
    <script src='@{optimizelyBaseUrl}@{id}.js' type='text/javascript'></script>
  }
```

### src/main/twirl/uk/gov/hmrc/play/views/layouts/head.scala.html
```
@optimizelySnippet()
```

### src/test/scala/uk/gov/hmrc/play/views/layouts/HeadSpec.scala
something like https://github.com/hmrc/play-ui/blob/master/src/test/scala/uk/gov/hmrc/play/views/layouts/BetaBannerSpec.scala